hack/update-helm-index.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -e
+
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o pipefail
+
+this=`basename $0`
+
+usage () {
+cat << EOF
+Usage: $this [-h] [--from-url URL] [--from-local PATH]
+
+Options:
+  --from-url URL        specify the URL to download the Helm chart(s) archive(s) from
+						all Helm chart archives with the .tgz extension will be downloaded
+						and added to the Helm repo
+  --from-local PATH     specify the local path to the Helm chart archive (tgz file)
+						only one chart archive can be specified
+  --help/-h             show this help and exit
+
+NOTE: only one of --from-url or --from-local can be specified
+
+Example:
+
+  $this --from-url https://github.com/NVIDIA/k8s-device-plugin/archive/refs/tags/nvidia-device-plugin-0.15.0.tgz
+
+EOF
+}
+
+#
+# Parse command line
+#
+asset_url=
+asset_local=
+while [[ $# -gt 0 ]]; do
+	key="$1"
+	case $key in
+		--from-url)
```

Comment:
Could we check for the `https://` | `http://` prefix instead of requiring that a user set this explicitly?

Reply:
sure, I was not really sure how to make this friendly and defaulted for flags, but we can make it nicer, thanks for the recommendation